platform/src/main/java/org/stellar/anchor/platform/config/PropertySep24Config.java

```diff
@@ -1,11 +1,23 @@
 package org.stellar.anchor.platform.config;
 
 import lombok.Data;
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
 import org.stellar.anchor.config.Sep24Config;
 
 @Data
-public class PropertySep24Config implements Sep24Config {
-  boolean enabled = false;
-  int interactiveJwtExpiration = 300;
-  String interactiveUrl = "NA";
+public class PropertySep24Config implements Sep24Config, Validator {
+  boolean enabled;
```

Comment:
I think we should default `enabled = false` because we don't even support SEP24 at the moment. WDYT?

Reply:
The default Java Boolean is `false`. I had removed `enabled = false` from all SEP config classes to make them consistent.